test/evergreen.yml

```diff
@@ -44,7 +44,41 @@ functions:
     command: shell.exec
     params:
       script: |
+        set -o errexit
+        set -o verbose
         git clone https://github.com/wiredtiger/mongo-tests
+  "fetch mongodb repo" :
+    command: shell.exec
+    params:
+      script: |
+        set -o errexit
+        set -o verbose
+        git clone https://github.com/mongodb/mongo
+  "merge mongodb wt" :
```

Comment:
Is this task correctly define to merge local WT changes to mongodb ?

Reply:
Since I am calling `get project` beforehand, should be fine, but double checking.

Reply:
I think it's fine. Though this function depends on both "wiredtiger" and "mongo" local trees are in place, the way you use it has mandates the dependencies. I'm not quite sure if it's possible to call another function in a given function in evergreen yaml. And does "import wiredtiger into mongo" feel a more descriptive function name?

Reply:
Thanks - I will change the name as well here.
Indeed, this function needs to have two repos checked out to work correctly, the script will exit if not the case since we added `set -o errexit`.